### DISCUSSION TITLE
Seeing same errors even after downgrading datree

### DISCUSSION THREAD

**Discussion by ee07dazn** [datreeio/datree#943] [Q&A]
Hi 

Recently, we started getting a bunch of new errors on policy etc. for example - `Ensure seccomp profile is set to docker/default or runtime/default`. Our default approach to downloading and running datree was following 
`curl https://get.datree.io | /bin/bash` - I believe since this downloads the latest version, the above policies got added there and hence now I am seeing this issue. 

Temporarily we wanted to downgrade so ran the following command, 
`curl -L https://github.com/datreeio/datree/releases/download/1.8.46/datree-cli_1.8.46_Linux_x86_64.zip` ...but it seems I still get the errors that i get from the latest version. Am I missing something here.

Can someone please help on how to downgrade properly? 🙏

**Comment by adifayer** [comment]
Hey there! @ee07dazn 
Datree's default policy is not immutable, meaning sometimes in released version the active rules consisting the policy get updated.
To create an immutable policy, you may create your own policy (via the UI or Policy-As-Code), and turn on rules according to your use case.
